packages/admin/src/common/configVerification/ConfigVerification.tsx

```diff
@@ -0,0 +1,40 @@
+import { useQuery } from "@apollo/client";
+import { Alert, Loading, MainContent } from "@comet/admin";
+import { ContentScopeInterface } from "@comet/cms-admin";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { brevoConfigCheckQuery } from "./ConfigVerification.gql";
+import { GQLBrevoConfigCheckQuery, GQLBrevoConfigCheckQueryVariables } from "./ConfigVerification.gql.generated";
+
+interface ConfigCheckProps {
+    scope: ContentScopeInterface;
+}
+
+export function ConfigVerification({ scope, children }: React.PropsWithChildren<ConfigCheckProps>): JSX.Element {
+    const { data, error, loading } = useQuery<GQLBrevoConfigCheckQuery, GQLBrevoConfigCheckQueryVariables>(brevoConfigCheckQuery, {
+        variables: { scope },
+        fetchPolicy: "network-only",
```

Comment:
Can we change this to `cache-and-network` for a faster response with slow connection if the api key is configured and was already loaded on another page?